lms/views/dashboard.py

```diff
@@ -0,0 +1,42 @@
+from pyramid.httpexceptions import HTTPFound
+from pyramid.view import view_config
+
+from lms.security import Permissions
+from lms.services.h_api import HAPI
+
+
+class DashboardViews:
+    def __init__(self, request) -> None:
+        self.request = request
+        self.h_api = request.find_service(HAPI)
+        self.assignment_service = request.find_service(name="assignment")
+
+    @view_config(
+        route_name="dashboard.launch.assignment",
+        permission=Permissions.DASHBOARD_VIEW,
+        request_method="POST",
+    )
+    def assignment_redirect_from_launch(self):
```

Comment:
I couldn't come up with a good name for this so I come up with a long verbose one.

This view just redirect to the other one. The final version should also set a cookie.